### PR TITLE
"Made a review" Trigger [PREMIUM-249]

### PR DESCRIPTION
**Pull request by websupporter**
## Description

_N/A_

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

https://github.com/mailpoet/mailpoet-premium/pull/798

## Linked tickets

[PREMIUM-249]

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[PREMIUM-249]: https://mailpoet.atlassian.net/browse/PREMIUM-249?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/css/src/components-automation-integrations/woocommerce.scss [hunk 1]
```diff
@@ -19,3 +19,10 @@
     background: transparent;
   }
 }
+
+.mailpoet-product-rating-range {
+  input {
+    margin: 0 5px;
+    max-width: 50px;
+  }
+}
```
Comment (JanJakes) [inline mailpoet/assets/css/src/components-automation-integrations/woocommerce.scss]: What are these styles for? If it's for something in premium, why not put them there?

**Review by JanJakes — COMMENTED** [review]
LGTM, the build seems to be failing.

File: mailpoet/assets/css/src/components-automation-integrations/woocommerce.scss (hunk 1 above)
Comment (websupporter) [reply mailpoet/assets/css/src/components-automation-integrations/woocommerce.scss]: Moved into Premium

**Review by JanJakes — DISMISSED** [review]

**Comment by websupporter** [comment]
rebased after comment trigger went into trunk

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**